### PR TITLE
Update apvendor packages

### PR DESCRIPTION
**Pull request by himabindu07**
## Details

Image Name | Old Tag | New Tag
-- | -- | -- | 
ap-blackbox-exporter | 0.24.0-7|0.25.0
ap-kibana| 8.13.4|  8.14.1
ap-grafana | 10.4.1 | 10.4.5
ap-configmap-reloader | 0.13.0 |0.13.1 
ap-curator | 8.0.14 | 8.0.15
ap-nats-server | 2.10.10 | 2.10.14

## Related Issues

https://github.com/astronomer/issues/issues/6305

## Checklist

- [x] version.txt was updated

### CODE REVIEW

**Comment by pgvishnuram** [comment]
elastic we are not going to upgrade to latest

**Comment by himabindu07** [comment]
> elastic we are not going to upgrade to latest

Can I remove those changes ??

**Comment by himabindu07** [comment]
> > elastic we are not going to upgrade to latest
> 
> Can I remove those changes ??

Removed the changes

File: nats-server/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ ARG BUILD_NUMBER=-1
 LABEL io.astronomer.docker=true
 LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 
-ENV NATS_SERVER 2.10.10
+ENV NATS_SERVER 2.10.17
```
Comment (pgvishnuram) [inline nats-server/Dockerfile:24]: can you downgrade this to 2.10.14

File: nats-server/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-2.10.10
\ No newline at end of file
+2.10.17
```
Comment (pgvishnuram) [inline nats-server/version.txt:1]: can you downgrade this to 2.10.14

File: nats-server/Dockerfile (hunk 1 above)
Comment (himabindu07) [reply nats-server/Dockerfile:24]: > can you downgrade this to 2.10.14

Sure, but the latest version is 2.10.17 right, any specific reason

Comment (danielhoherd) [reply nats-server/Dockerfile]: @pgvishnuram i agree with @himabindu07 that i'd like to know why we cannot upgrade to the latest.

**Comment by pgvishnuram** [comment]
releasing new ap-base

**Pull request merged**